php/php.api.phpmodule/src/org/netbeans/modules/php/api/validation/ValidationResult.java

```diff
@@ -80,6 +81,20 @@ public boolean hasErrors() {
         return new ArrayList<>(errors);
     }
 
+    /**
+     * Get the first error.
+     *
+     * @return the first error.
```

Comment:
Maybe to be added `or {@code null} if there are no errors`?

Reply:
Will fix. Thanks!

Reply:
Fixed.
https://github.com/apache/netbeans/compare/5685f82c7815b1441ffe2188fa7c327dc79c8685..6c2e311da592782e23c67563514a380699ed4314